added customized filtering examples in logging.rst (#4965)
scrapy/scrapy: docs/topics/logging.rst

If you run this spider again then INFO messages from
``scrapy.spidermiddlewares.httperror`` logger will be gone.

> You can also filter messages by content using a substring or

Reviewer: Without going into specifics in the examples below, as to keep the section short, I would still make references to the `LogRecord` class and the fact that it offers more possibilities for filtering beyond log messages, e.g.

```suggestion
You can also filter log records by :class:`~logging.LogRecord` data. For example, you can filter log records by message content using a substring or
```

> a regular expression. Create a ``logging.Filter`` subclass 

Reviewer: It would be best to link the first references to new API objects to their reference documentation, e.g.

```suggestion
a regular expression. Create a :class:`logging.Filter` subclass 
```

and equip it with a regular expression pattern to
filter out unwanted messages::

    import logging
    import re
    
    class ContentFilter(logging.Filter):
        def filter(self, record):

>             m = re.search (r'.*\d\d\d [Ee]rror, retrying.+', record.msg)
>             if m.group(0) is not None:
>                 return False

Reviewer: It’s not really important, but I would refactor this for it to look a bit more clean, e.g.

```suggestion
            match = re.search(r'\d{3} [Ee]rror, retrying', record.message)
            if match:
                return False
```

                
A project-level filter may be attached to the root 
handler created by Scrapy, this is a wieldy way to 
filter all loggers in different parts of the project

> (middlewares, spider, etc.). Alternatively, you may 
> choose a specific logger and filter it without
> affecting other loggers::

Reviewer: ```suggestion
(middlewares, spider, etc.)::
```
